packages/tizen_bundle/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: tizen_bundle
+description: Tizen bundle APIs.
+homepage: https://github.com/flutter-tizen/plugins
+repository: https://github.com/flutter-tizen/plugins/tree/master/packages/tizen_bundle
+version: 0.0.1
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  plugin_platform_interface: ^2.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+  tizen_interop:
+    git: https://github.com/flutter-tizen/tizen_interop
```

Comment:
It is discouraged to use git dependencies in published packages. We need to publish [tizen_interop](https://github.com/flutter-tizen/tizen_interop) first as I mentioned in email. cc @WonyoungChoi

Reply:
Dear @swift-kim, 
Should I wait until the tizen_interop is published?
Should I write the binding codes to implement the Bundle API?
I've checked other packages as like shared_preferenence and tizen_app_control APIs.
There are binding codes.

Reply:
We'll publish tizen_interop when https://github.com/flutter-tizen/tizen_interop/pull/13 is merged. Note that the existing packages such as shared_preferenences_tizen and path_provider_tizen depend on pure Dart FFI but not the tizen_interop package.

Reply:
Thanks for sharing.
Yes. I mean I want to know I should implement the bundle API using pure Dart FFI.
I'll wait until it's published.

Reply:
As long as tizen_interop works as expected without problem, no, you don't have to. We'll review your code soon.

Reply:
Thanks.